apps/vaporgui/VSection.cpp

```diff
@@ -46,7 +46,7 @@ QString VSection::_createStylesheet() const
     }
     )";
 #elif defined(WIN32)
-#error style missing for windows
+//#error style missing for windows
```

Comment:
Please un-delete this error

Reply:
I've removed the #elif for WIN32, so it's using the stylesheet for Linux.  Shown below:

![unnamed](https://user-images.githubusercontent.com/9522770/69998891-9c0f5080-1514-11ea-970a-b1f015bb65fc.png)

Reply:
Great, glad it works, thanks!